exporter/datadogexporter/metrics_exporter_test.go

```diff
@@ -515,21 +514,20 @@ func Test_metricsExporter_PushMetricsData_Zorkian(t *testing.T) {
 						"host":   "test-host",
 						"tags":   []interface{}{"env:dev"},
 					},
+					map[string]interface{}{
+						"metric": "otel.system.disk.in_use",
+						"points": []interface{}{[]interface{}{float64(0), float64(333)}},
+						"type":   "gauge",
+						"host":   "test-host",
+						"tags":   []interface{}{"env:dev"},
+					},
 					map[string]interface{}{
 						"metric": "otel.datadog_exporter.metrics.running",
 						"points": []interface{}{[]interface{}{float64(0), float64(1)}},
 						"type":   "gauge",
 						"host":   "test-host",
 						"tags":   []interface{}{"version:latest", "command:otelcol"},
 					},
-					map[string]interface{}{
-						"metric":   "system.disk.in_use",
-						"points":   []interface{}{[]interface{}{float64(0), float64(333)}},
-						"type":     "gauge",
-						"host":     "test-host",
-						"interval": float64(1),
```

Comment:
Interval has been omitted from the new code unintentionally, but this is fine because it was wrong anyway. Some metrics have different (or no interval at all), and the old code (removed in this PR) was just adding interval "1" to everything, including container metrics. Having no interval is better than having the wrong interval.

Reply:
This was a mistake, it should have no interval. I fixed this a while ago on the Agent but forgot to do it here apparently